doc/release-notes-31223.md

```diff
@@ -0,0 +1,6 @@
+P2P and network changes
+-----------------------
+When the `-port` configuration option is used, the default onion listening port will now
+be derived to be that port + 1 instead of being set to a fixed value.
+This re-allows setups with multiple local nodes using different `-port`, which
+would lead to a startup failure in v28.0 due to a port collision error. (#31223)
```

Comment:
I meant `-bind=` as a means to disable the onion bind on `127.0.0.1:8334` to avoid the port collision, but I typed `-bind=...=onion` instead :(

One can use both depending on what's needed: `-bind=1.2.3.4:5555 -bind=127.0.0.1:8335=onion`.